LICENSE

```diff
@@ -0,0 +1,42 @@
+/*
+//@HEADER
+// ************************************************************************
```

Comment:
This is kind of silly but for the sake of scripts that read license files, we should probably remove the @HEADER and the C++ comments